langchain/src/schema/index.ts

```diff
@@ -101,8 +101,19 @@ export interface StoredGeneration {
 
 export type MessageType = "human" | "ai" | "generic" | "system" | "function";
 
+export type MessageContent =
+  | string
+  | (
+      | {
+          type: "text" | "image_url";
+          text?: string;
+          image_url: string;
```

Comment:
It should be this I think

```suggestion
          image_url: string | { url: string };
```